### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by jetexe**
change requirements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
-    "name": "saitswebuwm/xlsx2csv",
+    "name": "jetexe/xlsx2csv",
```
Comment (michaeljs1990) [inline composer.json]: needs to switch back to upstream version.

File: README.md [hunk 2]
```diff
@@ -7,5 +7,6 @@ An XLSX to CSV parser
 
 ```php
 $converter = new \XlsxToCsv\XlsxToCsv($filename);
+$converter->sheetNomber = 1;
```
Comment (michaeljs1990) [inline README.md]: typo?

File: src/XlsxToCsv/XlsxToCsv.php [hunk 3]
```diff
@@ -1,150 +1,137 @@
...
 
 use \PclZip;
 
-class XlsxToCsv
-{
-    private $fileToConvert = null; 
+class XlsxToCsv {
+
+    private $fileToConvert = null;
+    public $sheetNomber    = 1;
```
Comment (michaeljs1990) [inline src/XlsxToCsv/XlsxToCsv.php]: typo?

File: src/XlsxToCsv/XlsxToCsv.php [hunk 4]
```diff
@@ -1,150 +1,137 @@
...
         ob_end_flush();
-        $z->close( $filename );
+        $z->close($filename);
 
-        $dir = getcwd();
-        $filename = '/tmp/bin/xl/worksheets/sheet1.xml';
-        $z = new \XMLReader();
-        $z->open( $filename );
+        $dir      = getcwd();
+        $filename = '/tmp/bin/xl/worksheets/sheet' . $this->sheetNomber . '.xml';
```
Comment (michaeljs1990) [inline src/XlsxToCsv/XlsxToCsv.php]: typo?

**Review by michaeljs1990 — COMMENTED** [review]
Thanks for the touchups, didn't have a ton of time to look this over can you add a description of what your changes to the file actually accomplishes? Also any reason for forking the code and adding your own license?

**Comment by jetexe** [comment]
oh, a make pull request only for composer.json file on required version of php zip, other files was added automatically. own composer packedge it's a temp design for my projects. I will make pull request after cleanup and testing. License added from template cos there no any license.

File: composer.json (hunk 1 above)
Comment (jetexe) [reply composer.json]: you need to discard this changes. It's temp design

File: README.md (hunk 2 above)
Comment (jetexe) [reply README.md]: this part(and others) help us to convert not only first page of xlsx

**Review by michaeljs1990 — APPROVED** [review]

**Comment by cjmaio** [comment]
Closing per @jetexe request in #2

**Pull request closed without merging**